models.js

```diff
@@ -456,7 +456,7 @@ var makeXPathModels = function(hashtagConfig) {
         };
         this.toHashtag = _combine;
         this.getChildren = function () {
-           return this.steps;
+           return [];
```

Comment:
This was implemented incorrectly as hashtags don't actually support children (predicates) at the moment